src/ansible_runner/cleanup.py

```diff
@@ -143,7 +151,7 @@ def cleanup_dirs(pattern, exclude_strings=(), grace_period=GRACE_PERIOD_DEFAULT)
     return ct
 
 
-def cleanup_images(images, runtime='podman'):
+def cleanup_images(images: list, runtime: str) -> int:
```

Comment:
```suggestion
def cleanup_images(images: list, runtime: str='podman') -> int:
```
Missing default value?

Reply:
Nope. I changed the calls to `cleanup_images()` to make sure this value is never `None` by using the default runtime at that point.

Reply:
Ok. Got it.